packages/module/src/ChatbotHeader/ChatbotHeader.scss

```diff
@@ -39,22 +54,24 @@
 // ============================================================================
 .pf-chatbot--fullscreen {
   .pf-chatbot__header {
-    padding: var(--pf-t--global--spacer--lg) var(--pf-t--global--spacer--4xl) var(--pf-t--global--spacer--lg)
-      var(--pf-t--global--spacer--4xl);
     background-color: var(--pf-t--global--background--color--primary--default);
   }
   .pf-chatbot__header__divider {
     display: none;
   }
+
+  .pf-chatbot__actions {
+    .pf-v6-c-menu-toggle.pf-m-secondary {
+      width: 200px;
```

Comment:
Is there any chance that this menu toggle could contain contents that'd be wider than 200px?

Reply:
That's a good callout - we don't need this anymore now that the default icon is smaller on the smaller chatbot sizes.